examples/nodejs/BUILD

```diff
@@ -9,14 +9,14 @@ load("@io_bazel_rules_docker//container:container.bzl", "container_image")
 # https://howtonode.org/hello-node
 container_image(
     name = "hello",
-    base = "//experimental/nodejs:nodejs10",
+    base = "//experimental/nodejs:nodejs10_debian10",
```

Comment:
Could you also parameterize this and test both `_debian9` and `_debian10`? I'm sure this will be immensely helpful to prevent breakage. For example, `/experimental/python3/BUILD` has

```bazel
[container_test(
    name = "python3" + distro_suffix + "_test",
    size = "medium",
    configs = ["testdata/python3.yaml"],
    image = ":python3" + distro_suffix,
) for distro_suffix in ("_debian9", "_debian10")]
```